pkg/proc/proc_test.go

```diff
@@ -2562,6 +2562,31 @@ func TestStepOutBreakpoint(t *testing.T) {
 	})
 }
 
+func TestStepOutBreakpointKeepsSteppingBreakpoints(t *testing.T) {
```

Comment:
This is testing the exact same code as TestNextBreakpointKeepsSteppingBreakpoints, remove.

Reply:
Done.